content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,140 @@
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options" style="width:100%" >}}
+
+You can filter Sessions using **facets**. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet" style="width:100%" >}}
+
+You are also able to filter by [view][6], as well as customize and pin views. This can be helpful in the case that you know where the issue lies.
+
+{{< img src="/getting_started/session_replay/pinned-views.png" alt="Views dropdown" style="width:100%" >}}
+
+## Examine a user journey
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event.
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey" style="width:100%" >}}
+
+You are able to filter the User Journey by **View**, **Action**, **Error**, and **Frustration Signal** by selecting **Events**. 
+
+{{< img src="/getting_started/session_replay/views-actions-errors.png" alt="Panel that allows for filtering by view, action, error, and frustration signal" style="width:100%" >}}
+
+While hovering over a particular time or view in the user journey, selecting **Details** opens a side panel where you can examine specific Core Web Vitals of the page, as well as other pertinent information, without leaving the replay.
+
+{{< img src="/getting_started/session_replay/details-panel.png" alt="Panel with additional details">}}
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Troubleshoot using Developer Tools
+
+Session Replay's [Browser Developer Tools][8] enable you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console" style="width:100%" >}}
+
+You can view any **Resources** associated with a view by selecting the Resources tab.
+
+{{< img src="/getting_started/session_replay/resources.png" alt="Resources associated with the view" style="width:100%" >}}
+
+**Traces** associated with a view can be found in the [Traces tab](#tracing-with-the-apm-integration).
```

Comment:
These are very close to "click print to print" type documentation 😂 . WHY would someone look at resources? what kind of info will they find there to help them troubleshoot? What should they look for? Ditto with Traces